tests/runner.py

```diff
@@ -450,6 +450,12 @@ def setUp(self):
       self.working_dir = tempfile.mkdtemp(prefix='emscripten_test_' + self.__class__.__name__ + '_', dir=self.temp_dir)
     os.chdir(self.working_dir)
 
+    # Include emscripten root for node module lookup (NODE_PATH may have other entries)
+    if os.environ.get('NODE_PATH'):
+      os.environ['NODE_PATH'] = os.environ['NODE_PATH'] + os.pathsep + path_from_root('node_modules')
+    else:
+      os.environ['NODE_PATH'] = path_from_root('node_modules')
```

Comment:
I just moved this code to test_sockets.py in #10266.  My apologies for the churn, I probably should have waited until you landed this change.